include/circt/Dialect/Seq/Seq.td

```diff
@@ -47,10 +50,31 @@ def CompRegOp : SeqOp<"compreg",
     Optional<I1>:$reset, Optional<AnyType>:$resetValue);
   let results = (outs AnyType:$data);
 
-  let printer = "return ::print(p, *this);";
+  let printer = "return ::print$cppClass(p, *this);";
   let parser = "return ::parse$cppClass(parser, result);";
 }
 
+def RegOp : SeqOp<"reg",
```

Comment:
It seems likely that clients will use this namespace and the sv namespace.  Thus RegOp will conflict and users would have to prefix namespaces.  While not horrible, it is annoying.  Firrtl works aroudn this by having FModule and FVector for names which conflict with other common dialects to make life a bit nicer.

Reply:
So maybe RegisterOp or SeqRegOp?